skema/program_analysis/comment_extractor/server.py

```diff
@@ -0,0 +1,86 @@
+from pathlib import Path
+from io import BytesIO
+from zipfile import ZipFile
+from typing import List, Union, Optional
+from fastapi import FastAPI, File, UploadFile
+
+import skema.program_analysis.comment_extractor.comment_extractor as comment_service
+from skema.program_analysis.comment_extractor.model import (
+    SingleFileCommentRequest,
+    SingleFileCommentResponse,
+    MultiFileCommentRequest,
+    MultiFileCommentResponse,
+    SupportedLanguageResponse,
+)
+
+SUPPORTED_FILE_EXTENSIONS = [".c", ".cpp", ".py", ".F", ".f", ".f95", ".for", ".m", ".r"]
```

Comment:
This is a super minor thing but would it make sense to maintain this SUPPORTED_FILE_EXTENSIONS list in terms of the EXTENSION_TO_LANGUAGE dictionary? Perhaps by computing the keys of the dictionary? It's a super minor thing but it would mean one less structure to manually update.

Reply:
I agree, that was kind of a last minute addition. I'm planning on updating that during the next sprint when we add support for comment unification.